src/components/integrated/OakLessonVideoTranscript/OakLessonVideoTranscript.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { ReactNode, useState } from "react";
+
+import { OakBox, OakFlex } from "@/components/base";
+import { OakCollapsibleContent, OakTertiaryButton } from "@/components/ui";
+
+type OakLessonVideoTranscriptProps = {
+  /**
+   * The transcript content
+   */
+  children: ReactNode;
+  /**
+   * The id of the collapsible content element. This is used to link the button to the content
+   */
+  id: string;
+  /**
+   * A control to toggle the video to display sign language
+   */
+  signLanguageControl?: ReactNode;
```

Comment:
The logic to toggle sign-language is deliberately not part of this component. It will have a bunch of moving parts that don't really belong here (availability of the signed version, toggle state, and toggle handler) — better to leave it up to the consumer than drill props through this component.

Reply:
I'd like to try and capture a heuristic for when we should pass logic up to the consumer and when not.